### PR TITLE
Change message about calling compute_bias() twice

### PR DESCRIPTION
**Pull request by v-jkegler**
Change message about calling compute_bias() twice

### CODE REVIEW

File: parser/src/constraint.rs [hunk 1]
```diff
@@ -158,7 +158,7 @@ impl Constraint {
         ensure!(!self.last_res.is_stop(), "compute_bias() called after stop");
         ensure!(
             self.step_arg.is_some(),
-            "commit_token() not called before compute_bias()"
+            "compute_bias() called twice without intervening commit_token()"
```
Comment (hudson-ai) [inline parser/src/constraint.rs:161]: Can you change this error message to reflect the (I assume new) function name "compute_mask"? There are a few occurrences of the name "compute_bias" before this too.

Comment (v-jkegler) [reply parser/src/constraint.rs:161]: Both a compute_mask() and a compute_bias() are present in the code, one calling the other.  You and @mmoskal should do whatever you think appropriate here.

Comment (v-jkegler) [reply parser/src/constraint.rs:161]: @hudson-ai If you haven't already, you might want to consult the chat for yesterday, where @mmoskal and I had a brief discussion of this message.  It was that discussion which motivated me to file this issue.

**Comment by mmoskal** [comment]
I have a bunch of other changes pending around this and will work more on this today. Will get back to this PR when I'm done!

File: parser/src/constraint.rs (hunk 1 above)
Comment (v-jkegler) [reply parser/src/constraint.rs:161]: @hudson-ai No problem.  This is a very easy call stack to get lost in!

**Comment by mmoskal** [comment]
the change is already applied, thanks!

**Pull request closed without merging**